Userland/Libraries/LibAccelGfx/Painter.cpp

```diff
@@ -150,6 +150,42 @@ void Painter::fill_rect(Gfx::FloatRect rect, Gfx::Color color)
     glDrawArrays(GL_TRIANGLE_FAN, 0, 4);
 }
 
+void Painter::draw_line(Gfx::IntPoint a, Gfx::IntPoint b, float thickness, Gfx::Color color)
+{
+    draw_line(a.to_type<float>(), b.to_type<float>(), thickness, color);
+}
+
+void Painter::draw_line(Gfx::FloatPoint a, Gfx::FloatPoint b, float thickness, Color color)
+{
+    auto midpoint = (a + b) / 2.0f;
+    auto length = AK::sqrt(AK::pow(b.x() - a.x(), 2.0f) + AK::pow(b.y() - a.y(), 2.0f));
```

Comment:
```suggestion
    auto length = a.distance_from(b);
```

Reply:
But tbh this whole block may get a lot nicer when using `Vector2`s, especially when we widen it's API a bit
Maybe `Point` should get a conversion function to that as well